frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -60,15 +60,30 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
   );
   const [downloading, setDownloading] = React.useState(false);
   const [downloadError, setDownloadError] = React.useState<Error | undefined>();
-  const { scrollToBottom, onMount, editorOptions } = useCodeEditorAsLogs();
   const [isDownloadDropdownOpen, setIsDownloadDropdownOpen] = React.useState(false);
   const { isSmallScreen } = useWindowResize();
+  const logViewerRef = React.useRef<{ scrollToBottom: () => void }>();
 
   React.useEffect(() => {
     if (!isPaused && logs) {
-      scrollToBottom();
+      if (logViewerRef && logViewerRef.current) {
+        logViewerRef.current.scrollToBottom();
+      }
     }
-  }, [isPaused, logs, scrollToBottom]);
+  }, [isPaused, logs]);
```

Comment:
Is this supposed to scroll to the bottom? It didn't work too well when I tested. The scroll stops somewhere before the end of logs for me. Here is a GIF. 

https://github.com/opendatahub-io/odh-dashboard/assets/22885912/4b243016-29e6-4491-a0ac-1cf694b972cc